### ISSUE TITLE
WER in report is much more than in each sentence

### ISSUE THREAD

**Issue by hao-olli-ai** [mozilla/DeepSpeech#1643]
`I Test of Epoch 7593 - WER: 0.001493, loss: 0.06935515210205888, mean edit distance: 0.000364                                                                                                                                                                 
I --------------------------------------------------------------------------------
I WER: 0.034483, loss: 0.733733, mean edit distance: 0.008197
I  - src: "VỚI VỞ KỊCH THỊ NỞ GẶP CHÍ PHÈO DIỆP ANH ĐÃ DIỄN RẤT ĐẠT VÀ GÂY ĐƯỢC CẢM TÌNH VỚI BAN GIÁM KHẢO CŨNG NHƯ ĐÔNG ĐẢO KHÁN GIẢ"
I  - res: "VỚI VỞ KỊCH THỊ NỞ GẶP CHÍ PHÈO DIỆP ANH ĐÃ DIỄN RẤT ĐẠT VÀ GÂY ĐƯỢC CẢM TÌNH VỚI BAN GÁM KHẢO CŨNG NHƯ ĐÔNG ĐẢO KHÁN GIẢ"
I --------------------------------------------------------------------------------
I WER: 0.035714, loss: 0.722248, mean edit distance: 0.008130
I  - src: "NẾU BẮT GẶP MƯỜI BÍ QUYẾT THÀNH CÔNG CỦA NGƯỜI DO THÁI HẲN TÔI SẼ CHỌN NGAY THÔI VÌ ĐÂY CŨNG LÀ THỂ LOẠI SÁCH TÔI YÊU THÍCH"
I  - res: "NẾU BẮT GẶP MƯỜI BÍ QUYẾT THÀNH CÔNG CỦA NGƯỜI DO THÁI HẲN TÔI SẼ CHỌN NGÀY THÔI VÌ ĐÂY CŨNG LÀ THỂ LOẠI SÁCH TÔI YÊU THÍCH"
I --------------------------------------------------------------------------------
I WER: 0.035714, loss: 0.766913, mean edit distance: 0.008850
I  - src: "DỌC BỜ BIỂN Ở BÃI BẮC CÓ RẤT NHIỀU DỨA NÚI MỘT LOẠI TRÁI CÂY CÓ MÙI THƠM ĐẶC BIỆT VÀ CHỈ CÓ Ở MIỀN BIỂN QUẢNG NAM"
I  - res: "DỌC BỜ BIỂN Ở BÃI BẮC CÓ RẤT NHIỀU DỨA NÚI MỘT LOẠI TRÁI CÂY CÓ MÙI THƠM ĐẶC BIỆT VÀ CHỈ CÓ Ở VIỀN BIỂN QUẢNG NAM"
I --------------------------------------------------------------------------------
I WER: 0.037037, loss: 0.760990, mean edit distance: 0.008547
I  - src: "SỬ DỤNG XUNG ĐỘT ĐỂ DẠY KỸ NĂNG GIẢI QUYẾT VẤN ĐỀ LUÔN LUÔN ĐẶT CHO TRẺ NHỮNG GIỚI HẠN VÀ HƯỚNG TRẺ TỚI MỘT GIẢI PHÁP"
I  - res: "SỬ DỤNG XUNG ĐỘT ĐỂ DẠ KỸ NĂNG GIẢI QUYẾT VẤN ĐỀ LUÔN LUÔN ĐẶT CHO TRẺ NHỮNG GIỚI HẠN VÀ HƯỚNG TRẺ TỚI MỘT GIẢI PHÁP"
I --------------------------------------------------------------------------------
`

Hi everyone,

After training DS for Vietnamese ASR, and I got the results below. 

Training set and validation set are the same (overfitting). I used a small set (4 hours) for them.

As can be seen, WER at epoch 7593 is 0.001943, but the printed WER for every sentences are more than 0.03. 

Could you help me to answer why the WER for an epoch is much more than WER for each sentences.

Thanks.

**Comment by lissyx** [comment]
@hao-olli-ai I think it's just because the sampled output is for the worst WER reported. It would suggest that you have a lot of much lower WER than 0.001943 :)
